pages/team.tsx

```diff
@@ -22,6 +30,27 @@ export default function Team() {
           />
         </Heading>
       </Flex>
+
+      <VStack>
+        {error != null
+          ? error.message
+          : members?.map((m) => (
+              <p key={m.id}>
+                {m.id} {m.name} {m.role}
+              </p>
+            ))}
+      </VStack>
     </Section>
   );
 }
+
+export const getStaticProps: GetStaticProps<Props> = async () => {
```

Comment:
esse get Static props nao poderia ser movido para outro arquivo? ou todas vez que ele tem que ser usado ele tem que perterncer ao arquivo cujo qual eles esta controlando a renderização ?

Reply:
Da uma lida nas referencias, principalmente no que diz sobre data fetching do next. Esse cara é exclusivo do next e tem que ficar na página que usa os dados.

- [Data Fetch and getStaticProps](https://nextjs.org/docs/basic-features/data-fetching#runs-on-every-request-in-development)